.travis.yml

```diff
@@ -7,9 +7,6 @@ os: osx
 osx_image: xcode6.4
 
 env:
-  matrix:
-    
-    - CONDA_R=3.3.2
```

Comment:
@johanneskoester I guess that this is not OK yet :unamused: at least it solved #1